src/components/atoms/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import "./BackButton.scss";
+
+interface BackButtonProps {
```

Comment:
```suggestion
export interface BackButtonProps {
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc